docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1695 @@
+{
```

Comment:
<p>nit: the linear predictor does not contain the error term (for reference, the paragraph between eqs 2 and 3 in the tutorial by Buerkner and Vuorre</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>